src/Process.js

```diff
@@ -77,10 +81,12 @@ Process.prototype.stream = T.func([], TDuplexStream, "process.stream").of(functi
 /**
  * Kills a running Process
  *
- * @param {String} signal to kill the process
+ * @param {MaybeExitCodes} maybeExitCodes that are valid
  */
-Process.prototype.kill = T.func([], T.Object, "process.kill").of(function() {
-    return this.instance.kill();
+Process.prototype.kill = T.func([MaybeExitCodes], T.Object, "process.kill").of(function(maybeExitCodes) {
```

Comment:
gibts denn jetz en test, wo `process.kill()` ohne exitcodes gecalled wird - weil soweit ich das hier sehe, sollte das mit dem von dir genannten typeerror failen? verstehe grad nich, welcher vorschlag das produzieren soll, weil wenn dei typisierung um die funktion weg is, kann eigtl kein error kommen